doajtest/testbook/dashboard/historical_stats.yml

```diff
@@ -0,0 +1,97 @@
+suite: Dashboard
+testset: Historical Statistics
+tests:
+  - title: Historical Statistics Display - Managing Editor with 3 groups
+    context:
+      role: managing editor/admin
+      testdrive: statistics
+    steps:
```

Comment:
I did not see a motivational banner on this test

Reply:
This is an error in the test, not the code. According to: https://github.com/DOAJ/doajPM/issues/3849#issuecomment-2100138455 :
> Managing Editors - partially correct. They don't need to see their own Accepted/rejected stats at the top

I will fix the test.